ivadomed/main.py

```diff
@@ -308,6 +308,10 @@ def run_command(context, n_gif=0, thr_increment=None, resume_training=False):
     """
     command = copy.deepcopy(context["command"])
     path_output = set_output_path(context)
+    log_file = os.path.join(context['path_output'], context['log_file'])
+    logger.remove()
```

Comment:
note this is to prevent multiple threads all registering the same sink multiple times (double/tripple logging)